stories/imdb/stories/filterList.js

```diff
@@ -108,7 +133,80 @@ export default () => (
                   path={['searchRoot', 'releases']}
                   format={formatYear}
                 />
-                <TermsStats path={['searchRoot', 'genreScores']} />
+                <TermsStats
+                  path={['searchRoot', 'genreScores']}
+                  layout="table"
+                  tableAttrs={{ style: { margin: 'auto' } }}
+                  columns={[
+                    {
+                      field: 'key',
+                      label: 'Genre',
+                    },
+                    {
+                      field: 'count',
+                      label: 'Found',
+                    },
+                    {
+                      field: 'key',
+                      label: '',
+                      details: {
+                        expand: {
+                          display: x =>
+                            `Show top 50 based on meta score for ${x} ▼`,
+                        },
+                        collapse: {
+                          display: x =>
+                            `Hide top 50 based on meta score for ${x} ▲`,
+                        },
+                        component(x) {
```

Comment:
I love how extensible this table component is!

Little question: Would there be a better name for this property? I get why it's called component, but we usually use component in uppercase! There's no actual need on keeping consistency here, but makes me wonder if we could make the difference clearer (if not, disregard!).

Reply:
made it look for capital C